services/121-service/src/transaction-jobs/processors/transaction-job-intersolve-visa.processor.spec.ts

```diff
@@ -1,8 +1,10 @@
 import { TestBed } from '@automock/jest';
 import { Job } from 'bull';
+import Redis from 'ioredis';
 
-import { TransactionJobProcessorIntersolveVisa } from '@121-service/src/transaction-job-processors/processors/transaction-job-intersolve-visa.processor';
-import { TransactionJobProcessorsService } from '@121-service/src/transaction-job-processors/transaction-job-processors.service';
+import { REDIS_CLIENT } from '@121-service/src/payments/redis/redis-client';
+import { TransactionJobProcessorIntersolveVisa } from '@121-service/src/transaction-jobs/processors/transaction-job-intersolve-visa.processor';
+import { TransactionJobProcessorsIntersolveVisaService } from '@121-service/src/transaction-jobs/services/transaction-jobs-intersolve-visa.service';
```

Comment:
@RubenGeo it seems here, you updated the filenames, but not the class names. Probably in more places.

Reply:
this is done now